### PR TITLE
Updating the client recipe to handle orgs first

### PR DESCRIPTION
**Pull request by luckymike**

### CODE REVIEW

**Comment by luckymike** [comment]
Ready for review! This refactor brought to you by the glory of good test coverage.

File: test/integration/data-bag/serverspec/default_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require_relative './spec_helper'
+
+# describe 'chef-server-populator-cookbook-upload' do
```
Comment (cwjohnston) [inline test/integration/data-bag/serverspec/default_spec.rb]: Is there a reason for us to retain this commented code?

**Comment by cwjohnston** [comment]
I think this looks pretty good. Do you think it would enhance test coverage if we were to use different org for the test fixtures? In that case we can override the defaults in the test-kitchen config, and validate that the expected effect is reflected in the outcome of our integration tests.

**Comment by luckymike** [comment]
That commented bit should go (but also should be included down the road), I'll remove. I want to expand the test coverage to cover multiple orgs, clients, and users, overriding the default is a good idea as well. Do you think we should have that in now, or can that be a later update?

**Comment by cwjohnston** [comment]
I think we're in a good place to merge this now and add additional test fixtures later as you suggest. :+1:

**Pull request merged**